src/pages/our-principles/_data.tsx

```diff
@@ -39,6 +40,7 @@ export const principles: Array<PrinciplesType> = [
             },
         ],
         img: Reliable,
+        img_alt: '_t_Deriv values reliability_t_',
```

Comment:
no need to translate `alt`

Reply:
Nikita is right; It's not necessary to translate `image_alt` but they are already translated in some components and it will improve our SEO as well. IMO we can keep the translated `alt` .